packages/amplify-nodejs-function-runtime-provider/src/index.ts

```diff
@@ -17,7 +17,7 @@ export const functionRuntimeContributorFactory: FunctionRuntimeContributorFactor
           value: 'nodejs',
           cloudTemplateValue: 'nodejs14.x',
           defaultHandler: 'index.handler',
-          layerExecutablePath: path.join('nodejs', 'node_modules'),
+          layerExecutablePath: path.join('nodejs'),
```

Comment:
Since it's just a single string now, we can remove the path call here and remove the path import